### ISSUE TITLE
Accessibility improvements needed

### ISSUE THREAD

**Issue by 57TsZUu9tN4** [aeternity/aepp-sdk-js#2132]
Have you considered accessibility? There are some WCAG compliance issues.

**Comment by davidyuk** [comment]
Which specific issues do you get? The problem is that this SDK library doesn't have a web interface, so I'm unsure how to improve accessibility then.
